KIF Tests/AccessibilityActivationTests_ViewTestActor.m

```diff
@@ -0,0 +1,32 @@
+//
+//  AccessibilityActivateTests_ViewTestActor.m
+//  KIF Tests
+//
+//  Created by Alex Odawa on 09/07/2024.
+//
+
+#import <KIF/KIF.h>
+
+@interface AccessibilityActivateTests_ViewTestActor : KIFTestCase
+@end
+
+
+@implementation AccessibilityActivateTests_ViewTestActor
+
+- (void)beforeEach
+{
+    [[viewTester usingLabel:@"Accessibility"] tap];
+}
+
+- (void)afterEach
+{
+    [[[viewTester usingLabel:@"Test Suite"] usingTraits:UIAccessibilityTraitButton] tap];
+}
+
+- (void)testAccessibilityActivate
+{
+    [[viewTester usingLabel:@"AccessibilityView"] performAccessibilityActivateWithExpectedResult:YES];
+    [[viewTester usingValue:@"Activated"] tryFindingView];
```

Comment:
I think maybe we want `waitForView` here if it might need some time to handle the accessibility activation event. The `tryFindingView` method will assert the state immediately without any additional runloop spins.

Reply:
smart thinking, added